pkg/localvolume/localvolume.go

```diff
@@ -0,0 +1,128 @@
+package localvolume
+
+import (
+	"context"
+	"errors"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	core "k8s.io/api/core/v1"
+	//storagev1 "k8s.io/api/storage/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type LocalVolumes interface {
+	New(name string, labels map[string]string, nodeSelectors map[string]string, path string, storage resource.Quantity) (LocalVolume, error)
```

Comment:
maybe write `labels, nodeSelectors map[string]string`, signature will be shorter, same you could do with path and move it to 2nd place since I think that's even better to first provide name, path, storage, then additional fields (ordered by importance :green_circle: )

Reply:
changed the order